src/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -367,7 +367,6 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
     <Reference Include="System.IO.FileSystem" />
-    <Reference Include="System.Linq" />
```

Comment:
This is not needed

Reply:
This is to avoid them to use `System.Linq` code again.